r/rsparkling/DESCRIPTION

```diff
@@ -9,10 +9,11 @@ Authors@R: c(
   person("Jakub", "Hava", role = c("aut"), email = "jakub@h2o.ai"),
   person("RStudio", role = c("cph")),
   person("H2O.ai", role = c("cph")))
-Depends:
+Imports:
     utils,
     sparklyr (>= 0.3),
     h2o (>= 3.8.2.9)
```

Comment:
Yes. This change is a matter if rsparkling is acting as API without the need to call h2o functions directly by user. If rsparkling can be used in client mode where it use only own methods then why force user to attach h2o to `searchpaths()`. This is more a good practice, some people say it is _anti-social_ as it can cause conflicts in downstream dependencies built on top of rspkarling. In this case it is pretty unlikely, so it isn't as important. Let me know if you want to move h2o and/or sparklyr to depends.